hikari/guilds.py

```diff
@@ -1438,6 +2922,121 @@ def channels(self) -> typing.Mapping[snowflakes.Snowflake, channels_.GuildChanne
 
         return self.app.cache.get_guild_channels_view_for_guild(self.id)
 
+    @property
+    def text_channels(self) -> typing.Mapping[snowflakes.Snowflake, channels_.GuildTextChannel]:
```

Comment:
@FasterSpeeding wanted to move away from hiding cache calls behind properties. Might be worth leaving this part out till we decide what to do

Reply:
Ah ok!